code/app/src/main/res/layout/fragment_request.xml

```diff
@@ -1,76 +1,63 @@
-                android:id="@+id/userAvatar"
-                android:layout_width="48dp"
-                android:layout_height="48dp"
-                android:layout_marginEnd="16dp"
-                android:src="@drawable/ic_person" />
-
-            <TextView
-                android:id="@+id/requestMessage"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:text="User is requesting to follow you"
-                android:textColor="@android:color/black"
-                android:textSize="16sp"
-                android:textStyle="bold"
-                tools:text="User is requesting to follow you" />
-        </LinearLayout>
-    </androidx.cardview.widget.CardView>
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    app:cardCornerRadius="16dp"
+    app:cardElevation="0dp"
+    android:backgroundTint="#FCE4FA"
+    >
 
     <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center_vertical"
         android:orientation="horizontal"
-        app:layout_constraintTop_toBottomOf="@id/requestCard"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent">
-
-        <com.google.android.material.button.MaterialButton
+        android:padding="16dp">
+
+        <ImageView
+            android:id="@+id/uid"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:src="@drawable/ic_person"
+            android:scaleType="centerCrop"
+            android:clickable="true"
+            android:focusable="true" />
+
+        <TextView
+            android:id="@+id/requestMessage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="12dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:text="User is requesting to follow you"
+            android:textColor="@android:color/black"
+            android:textSize="14sp" />
+
+        <Space
+            android:layout_width="0dp"
+            android:layout_height="1dp"
+            android:layout_weight="1" />
+
+        <Button
             android:id="@+id/acceptButton"
-            android:layout_width="64dp"
-            android:layout_height="64dp"
-            app:cornerRadius="32dp"
-            app:icon="@drawable/ic_tick"
-            android:contentDescription="Accept follow request"
-            android:layout_marginEnd="24dp" />
+            android:layout_width="32dp"
+            android:layout_height="38dp"
+            android:layout_marginEnd="8dp"
+            android:background="@drawable/ic_tick"
```

Comment:
The icons honestly don't follow the rest of the theme. We should stick with the material icon set.